client/scss/components/_skiplink.scss

```
@@ -10,7 +10,7 @@
   }
 
   &.button {
-    background: theme('colors.positive.100');
+    background: theme('colors.secondary.DEFAULT');
     border: theme('colors.positive.100');
```

As the border color previously matched the background color, let's change it to match the new background color:

```suggestion
    border: theme('colors.secondary.DEFAULT');
```

---

I have just committed this suggestion.